release-notes/10.0/preview/preview2/winforms.md

```diff
@@ -1,14 +1,26 @@
 # Windows Forms in .NET 10 Preview 2 - Release Notes
 
-.NET 10 Preview 1 includes new Windows Forms features & enhancements:
+.NET 10 Preview 2 includes new Windows Forms features & enhancements:
 
-- [Feature](#feature)
+- [Clipboard Code Sharing with WPF](#clipboard-code-sharing-with-wpf)
+- [New System.Design Editors](#new-systemdesign-editors)
+- [Quality Enhancements](#quality-enhancements)
 
 ## Windows Forms updates in .NET 10
 
 - [What's new in Windows Forms in .NET 10](https://learn.microsoft.com/dotnet/desktop/winforms/whats-new/net100) documentation.
 - [Issues List for Windows Forms in .NET 10 Preview 2](https://github.com/dotnet/winforms/issues?q=is%3Aissue%20milestone%3A%2210.0%20Preview1%22%20)
 
-## Feature
+## Clipboard Code Sharing with WPF
 
-This is about the feature
+ We are unifying how WinForms and WPF handle the Clipboard, refining behaviors introduced in Preview 1. Over the course of Preview 2 the WinForms team has refactored much of the Clipboard code to a new location in order to make it shareable with WPF. By sharing and consolidating Clipboard code across both frameworks, we aim to enhance consistency, stability, and overall reliability for .NET Windows Desktop scenarios. We have also continued to enhance the Clipboard APIs introduced in Preview 1 with tweaks to the behavior in certain edge-cases.
+
+## New System.Design Editors
+
+We’ve introduced several new editors in the `System.Design` namespace, giving developers more robust options for creating designer applications. New editors include `ToolStripCollectionEditor` and several editors related to the `DataGridView` control.

```

Comment:
```suggestion
We’ve ported several UITypeEditors in the `System.Windows.Forms.Design` namespace from the .NET Framework code base, giving developers more robust options for creating designer applications. New editors include `ToolStripCollectionEditor` and several editors related to the `DataGridView` control. These editors will now be discoverable by the PropertyGrid and Designer Actions Panel. 

```
These editors are not public, I wanted to explain why they are not visible in the public surface diff.